src/components/Episode/Episode.jsx

```diff
@@ -53,6 +44,17 @@ export default function Episode(props){
 					thumbnail={ thumbnail }
 					trailer={ trailer }
 				/>
+				<div className={ s.content }>
+					<h2 className={ s.episode }>
+						Episode { number }
+					</h2>
+					<h1 className={ s.title }>
+						{ title }
+					</h1>
+					<p className={ s.body }>
```

Comment:
```suggestion
					<div className={ s.body }>
```
As we've done on the others - let's make the body container its own thing so that we can call `RENDER.body` and fill it with multiple `<p>` tags if needs-be.

Reply:
done!

Reply:
Pulled over the most up to date renderBody function from layout, and also corrected a small issue where an empty anchor would be included if the findLink function returned false.